src/Form/BasicSetup.php

```diff
@@ -134,15 +133,15 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           $context['setup_context'] = ['plugin_definition' => $plugin_definition];
           $form['#title'] = t('TFA setup - Trusted browsers');
           $setup_plugin = new TfaTrustedBrowserSetup($context);
-          $tfa_setup = new TfaSetup($setup_plugin, $context);
+          $tfa_setup = new TfaSetup($setup_plugin);
```

Comment:
If we removed $context, why are we building it above? see line 133 as an example